content/60.methods.md

```diff
@@ -231,30 +236,31 @@ Information about vaccine distribution was extracted from Our World In Data (<ht
 The Manubot framework facilitated a massive collaborative review on an urgent topic.
 This project demonstrates that Manubot can be applied to projects where not all collaborators have expertise or even experience working with version control pipelines.
-Through the development of cyberinfrastructure both for training novice users to interact with GitHub and simplifying the workflows to allow them to receive many of the benefits of What You See Is What You Get platforms such as Google Docs, we were able to adapt a powerful open publishing tool to harness the domain expertise of a large group of non-technical users and to respond to the flood of COVID-19 publications.
+This effort has produced not only seven literature reviews on topics relevant to the COVID-19 pandemic, but has also generated cyberinfrastructure for training novice users to interact with GitHub and extended functionalities of Manubut to provide more of the benefits of What You See Is What You Get platforms such as Google Docs.
+As a result, this project demonstrates the potential for open publishing to harness the domain expertise of a large group of non-technical users and to respond to the flood of COVID-19 publications.
 
 While Manubot manuscripts are written in markdown, they can be rendered in several formats that provide different advantages.
 For example, beyond building just a PDF, Manubot also renders the manuscript in HTML, DOCX, and now, LaTeX (in a more limited capacity).
 The interactive HTML manuscript format offers several advantages over a static PDF to harmonize available resources that we were able to apply to specific problems of COVID-19.
 The integration of scite into the HTML build makes references more manageable by visually representing whether their results are contested or whether they have been corrected or retracted.
 Cross-referencing different pieces of the manuscript, such as cited preprints with reviews stored in an appendix, is another interactive option presented by HTML.
 Additionally, because of the heavy emphasis on Word processing in biology, Manubot's ability to generate DOCX outputs was expanded to allow users to generate DOCX files containing only a section of the manuscript.
-In our case, where the full project is nearly 100,000 words, this allows individual pieces to be shared widely.
+In our case, where the full project is nearly 100,000 words, this allows individual pieces to be shared more easily.
 Finally, the preliminary addition of LaTeX output is useful for researchers from computational fields who submit papers in TeX format and removes the step of reformatting markdown prior to submission. 
 
+<!-- To Do: Add table listing additions to manubot-->
```

Comment:
Here is an initial summary:

- CI scripts to regularly download external data sources, generate figures and statistics, and store them in a GitHub branch that is read when Manubot builds the latest manuscript
- Zotero extension to report more relevant clinical trial metadata from https://clinicaltrials.gov
- Manubot feature to support citing any Compact Uniform Resource Identifier, such as `clinicaltrials` or `ncbigene`
- Integration with scite to track retractions, corrections, and notices of concern
- CI spell-checking reports that post as pull request comments
- Improved support for Pandoc's LaTeX output
- Building individual sections of a larger manuscript as standalone documents alongside the complete manuscript

This feels redundant with the existing text.  Do we agree with the reviewer that the summary is helpful?

Reply:
Yours is better than mine! I do like the idea of a table, because I have to keep looking through the text to jog my memory about the full list!